pallets/pallet-bonded-coins/src/lib.rs

```diff
@@ -850,6 +850,7 @@ pub mod pallet {
 				.ok_or(ArithmeticError::Overflow)?;
 
 			let real_costs = normalized_costs
+				// TODO: can easily overflow, causing test burn_large_quantity to fail
```

Comment:
This suffers from the same issues that convert_to_fixed suffered from: If we multiply then convert, this can easily overflow, but the other way around results in truncation. Should be solved just like the scaled conversion to fixed I'd say.